charts/hono/values.yaml

```diff
@@ -925,6 +899,105 @@ deviceRegistryExample:
     #  keyPath: "/etc/config/key.pem"
     #  certPath: "/etc/config/cert.pem"
 
+  # fileBasedDeviceRegistry contains configuration properties specific to the
+  # file based device registry.
+  fileBasedDeviceRegistry:
+    # imageName contains the name (including registry and tag)
```

Comment:
it doesn't contain the tag, does it?

Reply:
I have change it now and pushed a commit to address it and also the other comments. Would you mind taking a look again?